router/router.go

```diff
@@ -171,6 +172,14 @@ func Load(middleware ...gin.HandlerFunc) http.Handler {
 		debugger.GET("/pprof/trace", debug.TraceHandler())
 	}
 
+	monitor := e.Group("/api/metrics")
+	{
+		monitor.GET("/prometheus",
```

Comment:
let's use `/metrics` since it seems pretty standard. We can always change later if we decide we want to use a different endpoint.